src/pure.js

```diff
@@ -3,12 +3,16 @@ import {
   getQueriesForElement,
   prettyDOM,
 } from '@testing-library/dom'
+import { VERSION as SVELTE_VERSION } from 'svelte/compiler'
 import * as Svelte from 'svelte'
 
-const IS_SVELTE_5 = typeof Svelte.createRoot === 'function'
+const IS_SVELTE_5 = /^5\./.test(SVELTE_VERSION)
 const targetCache = new Set()
 const componentCache = new Set()
 
+if (IS_SVELTE_5)
+  console.warn('for Svelte 5, use `@testing-library/svelte/svelte5`')
```

Comment:
As mentioned in the previous comment, it's to deal with `import from 'svelte/legacy'`. We could probably dig out darker magic to do it another way, but I do feel there will be other things in Svelte 5 that will be easier if we just have different files implementing the same(ish) interface rather than having a file that tries to satisfy all versions at once.

Reply:
Doh, yeah duh, my bad, that makes total sense.

---
Aside: this makes me think dropping 3.0 in the `next` branch (in a separate PR to be responsible about it) will make things easier.

I'm not totally sure what to do about collecting multiple breaking changes without a prerelease version. Maybe we should switch over to an `beta` branch for breaking changes? Or consider switching `next` to be a prerelease in `semantic-release`.